src/rendering/renderers/shared/GenerateTextureSystem.ts

```diff
@@ -0,0 +1,139 @@
+import { extensions, ExtensionType } from '../../../extensions/Extensions';
+import { Matrix } from '../../../maths/Matrix';
+import { Rectangle } from '../../../maths/shapes/Rectangle';
+import { Bounds } from '../../scene/bounds/Bounds';
+import { getLocalBounds } from '../../scene/bounds/getLocalBounds';
+import { Container } from '../../scene/Container';
+import { RenderTexture } from './texture/RenderTexture';
+
+import type { Renderer } from '../types';
+import type { System } from './system/System';
+import type { TextureSourceOptions } from './texture/sources/TextureSource';
+import type { Texture } from './texture/Texture';
+
+// legacy support for v7
+export interface IGenerateTextureOptions
```

Comment:
I'm happy to have a breaking change in the types for `renderer.generateTexture` so i don't think we need this interface

We just replace it with ` GenerateTextureOptions | Container`

Reply:
Agree!